### ISSUE TITLE
[calendar] Implement add event UI 

### ISSUE THREAD

**Issue by noahmay** [Dsek-LTH/member-page#76]
## Short description of the feature
Implement add event UI 

Do this issue depend on another issue? Write Depends on #37

[x] Relates to frontend
[ ] Relates to the backend
[ ] Requires modification of GraphQL
[ ] Requires modification of Database
[ ] Requires creating a new service
